### PR TITLE
njsden TO file

### LINKED ISSUE #9: njsden's checklist
# Njsden's checklist

- [x] Create branch
- [ ] Work locally
- [ ] Create commit(s)
- [ ] Create pull request
- [ ] Collaborate
- [ ] Merge

### PR DESCRIPTION
**Pull request by njsden**
Creating a list of places to eat in TO.

Closes #9

### CODE REVIEW

File: njsden-to.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Places to eat
+
+- Wendy's
+- Pickle Barrel
```
Comment (githubteacher) [inline njsden-to.md]: Never heard of Pickle Barrel, what should I get there?

File: njsden-to.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# Places to eat
+
+- Wendy's
```
Comment (githubteacher) [inline njsden-to.md]: I wonder if there are items **not** on the 🇺🇸 menu

**Review by mraizman — COMMENTED** [review]
Thank you!

**Review by githubteacher — COMMENTED** [review]
Just a couple comments on your file.

**Comment by githubteacher** [comment]
@njsden I really like 🌮 s and 🍺 where should I go?

File: njsden-to.md (hunk 1 above)
Comment (njsden) [reply njsden-to.md]: All sorts of food (hamburgers, steak, seafood, salads, etc).

**Comment by njsden** [comment]
> @njsden I really like :taco: s and :beer: where should I go?

@beardofedu Mexican Amigos is a pretty decent place (in Richmond Hill though)!

**Pull request merged**